### ISSUE TITLE
Fix duplicate NAME and MANAGED_BY relationships

### ISSUE THREAD

**Issue by m-appel** [InternetHealthReport/internet-yellow-pages#22]
Somewhere in the [PeeringDB](../tree/main/iyp/crawlers/peeringdb) crawlers, a lot of duplicate relationships with exactly the same data are created, which should not be the case.

![graph](https://user-images.githubusercontent.com/49444122/213420187-ef83106d-5272-4a8f-bc84-86690e2a5376.svg)

Generated by
```cypher
MATCH (a:AS {asn: 2497})-[nr:NAME]->(n:NAME {name: 'Internet Initiative Japan'}),
  (a)-[m:MANAGED_BY]->(o:ORGANIZATION {name: 'Internet Initiative Japan Inc. (IIJ)'})
RETURN a,nr,n,m,o
```

**Comment by romain-fontugne** [comment]
I'll try to fix this one. I think the problem is coming from iyp/crawlers/peerindb/ix.py

**Comment by romain-fontugne** [comment]
fixed in ead5537237eb0ebca19047a86d1b5dfe261977de
